### PR TITLE
Optimize command examples in Lisk core v4.0.0

### PR DESCRIPTION
**Pull request by ManuGowda**
Closes https://github.com/LiskHQ/lisk-core/issues/796

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/core-cli.adoc
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/core-cli.adoc:1]: Haven't check the changes in this file assuming it was just copy pasted correctly from the source.

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 1]
```diff
@@ -91,20 +91,27 @@ Please refer to {url_docker_install_windows}[Docker installation for Windows^].
...
+git clone https://github.com/LiskHQ/lisk-core.git
+----
 
+.Navigate into the `lisk-core` root folder
 [source,bash]
 ----
-git clone https://github.com/LiskHQ/lisk-core.git #<1>
-cd lisk-core/docker                  #<2>
-git checkout v4.0.0 -b v4.0.0 #<3>
+cd lisk-core/docker
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:107]:
```suggestion
cd lisk-core
```

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 2]
```diff
@@ -91,20 +91,27 @@ Please refer to {url_docker_install_windows}[Docker installation for Windows^].
...
 ----
 
-<1> Clone the repository.
-<2> Navigate into the `lisk-core` root folder.
-<3> Check out the latest release tag.
+.Check out the latest release tag
+[source,bash]
+----
+git checkout v4.0.0 -b v4.0.0
+----
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:114]:
```suggestion
.Check out the latest release tag
[source,bash]
----
git checkout v4.0.0
----

.Build the docker image
[source,bash]
----
make build-image
----
```

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 3]
```diff
@@ -161,30 +161,56 @@ npm install pm2 -g
...
 ----
 
-<1> Clone the repository.
-<2> Navigate into the `lisk-core` root folder.
-<3> Check out the latest release tag.
-<4> Install dependencies.
+.Navigate into the `lisk-core` root folder
+[source,bash]
+----
+cd lisk-core/docker
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/source.adoc:175]:
```suggestion
cd lisk-core
```

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 4]
```diff
@@ -161,30 +161,56 @@ npm install pm2 -g
...
+----
 
-NOTE: Please check for the latest release in {url_core_releases}[core releases^].
+.Check out the latest release tag
+[source,bash]
+----
+git checkout v4.0.0 -b v4.0.0
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/source.adoc:181]:
```suggestion
git checkout v4.0.0
```

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 5]
```diff
@@ -161,30 +161,56 @@ npm install pm2 -g
...
+[source,bash]
+----
+yarn install --frozen-lockfile
+----
+.Build the binary
+[source,bash]
+----
+yarn build
+----
+To set up the Lisk core alias, add the following line to your `~/.profile` and run `source ~/.profile`.
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/source.adoc:195]:
```suggestion
Set up the `lisk-core` alias: Append the following line to your `~/.profile` file and run `source ~/.profile`.
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: docs/antora/modules/ROOT/pages/core-cli.adoc
Comment (chris529) [reply docs/antora/modules/ROOT/pages/core-cli.adoc:1]: Yes, you are correct 👍

**Review by sameersubudhi — APPROVED** [review]

File: docs/antora/modules/ROOT/pages/core-cli.adoc [hunk 6]
```diff
@@ -27,7 +27,7 @@ VERSION
   lisk-core/4.0.0 darwin-arm64 node-v18.16.0
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/core-cli.adoc:27]: - In the examples tab of each command, there is a `$` sign at the start, please remove that.
- Please also separate commands from the response, as suggested in the issue's description.

File: docs/antora/modules/ROOT/pages/core-cli.adoc [hunk 7]
```diff
@@ -335,7 +355,7 @@ Commands relating to Lisk Core blockchain data.
 [source,bash]
 ----
 USAGE
-  $ lisk-core blockchain:COMMAND
+  lisk-core blockchain
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/core-cli.adoc:358]: I see a `$` sign at the start in some places, and in others, I don't. Do you know if this is an inconsistency from SDK?

File: docs/antora/modules/ROOT/pages/management/configuration.adoc [hunk 8]
```diff
@@ -68,10 +70,38 @@ Use the dedicated flags to enable them on the application start:
...
+      "clearBlockHeadersInterval": 60000,
+      "encryptedPrivateKey": "kdf=argon2id&cipher=aes-128-gcm&version=1&ciphertext=aa92aefc87797b8e107e0ebc8f863333b8dbc173b7d6e75b0f3ca1f5113d64050247a2c79679840193ea564a34c80f7e3d8ff77d5a9a2cd7e9c1e912e707b38206d4700485013ae14e33db2e31f4f0cfeee5fdbe4e2356cce179819f87da348cf2fe057f6e320b809db3ffdad7cbae86c568184caebe3ba44edb49c59e263012&mac=e4d31ce6274e3dd83c70732f9965846114d53e07e831b1f5bb9a841e1731d85a&salt=a3fd8ab81c91b9b5&iv=c3028fe54417f71e1caba5727be0040d&tag=f97a0df7b6349f90310ebf3eed4dde42&iterations=1&parallelism=4&memorySize=2097023",
+      "fee": 100000000
+    }
+  }
+}
+----
+
+
+// Alternatively, enable the plugins directly in the config file, see <<using-a-custom-config-file>>.
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/management/configuration.adoc:104]: The entire process of enabling a report misbehavior plugin has already been explained in a separate guide here: https://lisk.com/documentation/beta/run-blockchain/enabling-misbehavior-report.html

This is redundant content and will affect docs maintainability. Please remove this and simply refer to the existing guides about enabling different plugins. Just refer to the following two guides.

- https://lisk.com/documentation/beta/run-blockchain/enabling-misbehavior-report.html
- https://lisk.com/documentation/beta/build-blockchain/using-dashboard.html

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 9]
```diff
@@ -91,20 +91,33 @@ Please refer to {url_docker_install_windows}[Docker installation for Windows^].
...
+The following commands below describe how to clone the repository, then how to navigate into the `lisk-core` root folder, and how to check out the latest release tag.
 
-== Lisk Core download
+.Clone the repository
+[source,bash]
+----
+git clone https://github.com/LiskHQ/lisk-core.git
+----
 
+.Navigate into the `lisk-core` root folder
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:104]: There is an issue with rendering monospace in captions, please either use quotation marks or write without them.
```suggestion
.Navigate into the "lisk-core" root folder
```

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 10]
```diff
@@ -161,30 +161,56 @@ npm install pm2 -g
...
-git checkout v4.0.0 -b v4.0.0                     #<3>
-npm install --global                              #<4>
+git clone https://github.com/LiskHQ/lisk-core.git
 ----
 
-<1> Clone the repository.
-<2> Navigate into the `lisk-core` root folder.
-<3> Check out the latest release tag.
-<4> Install dependencies.
+.Navigate into the `lisk-core` root folder
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/setup/source.adoc:172]:
```suggestion
.Navigate into the "lisk-core" root folder
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Please consider the following!

File: docs/antora/modules/ROOT/pages/core-cli.adoc (hunk 6 above)
Comment (chris529) [reply docs/antora/modules/ROOT/pages/core-cli.adoc:27]: Done 👍

File: docs/antora/modules/ROOT/pages/core-cli.adoc (hunk 7 above)
Comment (chris529) [reply docs/antora/modules/ROOT/pages/core-cli.adoc:378]: Now all descriptions (USAGE), start with the $ sign to maintain uniformity throughout the page 👍

File: docs/antora/modules/ROOT/pages/management/configuration.adoc (hunk 8 above)
Comment (chris529) [reply docs/antora/modules/ROOT/pages/management/configuration.adoc]: Good point..now done 👍

File: docs/antora/modules/ROOT/pages/core-cli.adoc [hunk 11]
```diff
@@ -27,7 +27,7 @@ VERSION
   lisk-core/4.0.0 darwin-arm64 node-v18.16.0
```
Comment (TalhaMaliktz) [inline docs/antora/modules/ROOT/pages/core-cli.adoc:28]: **Line 8**
`:source-language: bash`

**Review by TalhaMaliktz — APPROVED** [review]
Just a small addition required! rest LGTM 👍🏼

File: docs/antora/modules/ROOT/pages/setup/source.adoc [hunk 12]
```diff
@@ -161,30 +161,56 @@ npm install pm2 -g
...
+.Build the binary
+[source,bash]
+----
+yarn build
+----
+Set up the `lisk-core` alias: Append the following line to your `~/.profile` file and run `source ~/.profile`.
+[source,bash]
+----
+alias lisk-core=/home/lisk/lisk-core/bin/run
+----
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/source.adoc:199]:
```suggestion
Set up the `lisk-core` alias: Append the following line to your `~/.profile` file and run `source ~/.profile`.
[source,bash]
----
alias lisk-core=/home/lisk/lisk-core/bin/run
----
NOTE: In the command above, please replace `/home/lisk/lisk-core` with the absolute path to your "lisk-core" directory. To find the exact path on your system, change to your lisk-core directory and run the `pwd` command.
```

**Pull request merged**